lib/assertions.js

```diff
@@ -1355,16 +1357,7 @@ module.exports = expect => {
   );
 
   expect.addAssertion(
-    '<UnexpectedError> to [exhaustively] satisfy <function>',
-    (expect, subject, value) =>
-      expect.promise(() => {
-        subject.serializeMessage(expect.outputFormat());
```

Comment:
I'm not sure we can cut this out. If we would like the user to be able to assert on the error message in the function. I think we should make a special case:

```js
expect.addAssertion(
  '<UnexpectedError> to [exhaustively] satisfy <expect.it>', (expect, subject, value) => 
    expect.promise(() => {
      subject.serializeMessage(expect.outputFormat());
      return value(subject);
    })
)
```

Reply:
Mhh this test indicate something differently: This test indicates something differently https://github.com/unexpectedjs/unexpected/blob/feature%2FfunctionSubtypeOfObject/test/assertions/to-throw.spec.js#L12

Let's just let this be handled on the user side. Serializing the message to the wrong format might not be what you always want.